tgui/packages/tgui/interfaces/PaperSheet.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @file
+ * @copyright 2020 WarlockD (https://github.com/warlockd)
+ * @author Original WarlockD (https://github.com/warlockd)
+ * @author Changes stylemistake
+ * @license MIT
+ */
+
+import { Tabs, Box, Flex, Button, TextArea } from '../components';
+import { useBackend, useSharedState, useLocalState } from '../backend';
+import { Window } from '../layouts';
+// import marked from 'marked';
+import marked from 'marked';
+import DOMPurify from 'dompurify';
+// There is a sanatize option in marked but they say its deprecated.
+// Might as well use a proper one then
+
+import { createLogger } from '../logging';
+import { Fragment } from 'inferno';
+
+const logger = createLogger('PaperSheet');
+
+const run_marked_default = value => {
+  const sanitizer = DOMPurify.sanitize;
+  // too much?
+  // return sanitizer(marked(sanitizer(value),
+  //   { breaks: true, smartypants: true });
+  return sanitizer(marked(value,
+    { breaks: true, smartypants: true }));
```

Comment:
```suggestion
    { breaks: true, smartypants: true, sanitize: true }));
```
I recommend turning it's own sanitizer on.